reconstruct/reconstruct.go

```diff
@@ -3,49 +3,49 @@ package reconstruct
 import (
 	"fmt"
 	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/expandedTree"
 	"github.com/vertgenlab/gonomics/fasta"
-	"github.com/vertgenlab/gonomics/tree_newick"
 )
 
 //final function to run
-func Reconstruct(root *tree_newick.NTree, filename_output string) {
-	leaf := Get_leaf(root)
-	branches := Get_branch(root)
+func Reconstruct(root *expandedTree.ETree, outFilename string) {
+	leaf := expandedTree.GetLeaf(root)
+	branches := expandedTree.GetBranch(root)
 	length := len(leaf[0].Fasta.Seq)
 
 	for i := 0; i < length; i++ {
 		//set up tree
-		Set_state(root, i)
+		SetState(root, i)
 		Postorder(root)
 		// loop nodes to get probable base at that site and node
 		Loop_nodes(root)
 	}
 	var fastas []*fasta.Fasta
 
-	for i := 0; i < len(branches); i++ {
-		fastas = append(fastas, branches[i].Fasta)
+	for j := 0; j < len(branches); j++ {
+		fastas = append(fastas, branches[j].Fasta)
 	}
 
 	var fas []*fasta.Fasta
 	//loop sites
-	for i := 0; i < len(fastas); i++ {
-		fas = append(fas, fastas[i])
+	for k := 0; k < len(fastas); k++ {
```

Comment:
I am having trouble seeing what this loop does.  In the previous loop we went over all the nodes and added the sequence at that node to a slice.  It looks like this now goes over that slice and makes a new one that is equal to the one we already had?  Seems like we could remove "fas" and this loop and just write fastas instead of fas